### PR TITLE
[ADD] sawb_custom_hide_vat_for_individual

### PR DESCRIPTION
**Pull request by victor-champonnois**
## Description
Hide VAT for individuals

## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#view_type=form&model=project.task&id=10180&active_id=10180&menu_id=

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: sawb_custom_hide_vat_for_individual/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "website": "https://coopiteasy.be",
+    "author": "Coop IT Easy SC",
+    "maintainers": ["victor-champonnois"],
+    "license": "AGPL-3",
+    "application": False,
+    "depends": [],
+    "excludes": [],
+    "data": ["views/res_partner_view.xml"],
+    "demo": [],
+    "qweb": [],
```
Comment (huguesdk) [inline sawb_custom_hide_vat_for_individual/__manifest__.py]:
```suggestion
    "depends": [
        "base",
    ],
    "data": [
        "views/res_partner_view.xml",
    ],
```
please [avoid empty keys](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6) and use one line per list element like in [the example manifest](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py).

edit: added dependency to `base`.

File: sawb_custom_hide_vat_for_individual/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<odoo>
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">view_partner_form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="vat" position="attributes">
+                <attribute
+                    name="attrs"
+                >{'invisible': [("is_company", "=", False)]}</attribute>
```
Comment (huguesdk) [inline sawb_custom_hide_vat_for_individual/views/res_partner_view.xml:11]: just a rant against odoo: good variable naming and design would make the condition much more understandable: `{'visible': [("is_company", "=", True)]}`. general dev rule: always use positive names for flag attributes.

**Review by huguesdk — APPROVED** [review]

File: sawb_custom_hide_vat_for_individual/__manifest__.py (hunk 1 above)
Comment (victor-champonnois) [reply sawb_custom_hide_vat_for_individual/__manifest__.py]: Noted, thanks !

File: sawb_custom_hide_vat_for_individual/views/res_partner_view.xml (hunk 2 above)
Comment (victor-champonnois) [reply sawb_custom_hide_vat_for_individual/views/res_partner_view.xml:11]: Good point, I always spend a few seconds computing the condition in my head ^^

**Pull request merged**